src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,87 @@
+package main.java.duke.task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents the task list contains the user's tasks.
+ */
+public class TaskList {
+    List<Task> taskList;
```

Comment:
Might want to use "tasks" here to indicate plurality as tasklist was used previously.